test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
No. Absolutely not. I'm not saying mix tabs and spaces in the same file (or even the same project). I'm talking about working on a different project (maybe one you pulled from github) and that uses tabs.
Again, I have a global .jscsrc file (in my home dir), so my linter settings take over for this project I pulled.